src/lib/jar.js

```diff
@@ -9,41 +10,66 @@ var xhr = require('xhr');
  * set(name, value) – synchronously sets the cookie
  * use(name, uri, callback) – can by sync or async, gets cookie from the uri if not there.
  */
-var Jar = {};
+var Jar = {
+    unsign: function (value, callback) {
+        // Return the usable content portion of a signed, compressed cookie
+        if (!value) return callback('No value to unsign');
+        try {
+            var b64Data = value.split(':')[0];
+            var decompress = false;
+            if (b64Data[0] === '.') {
+                decompress = true;
+                b64Data = b64Data.substring(1);
+            }
 
-Jar.get = function (name, callback) {
-    // Get cookie by name
-    var obj = cookie.parse(document.cookie) || {};
+            // Django makes its base64 strings url safe by replacing + and / with - and _ respectively
```

Comment:
Wondering if we should drop in a link to Django's docs and/or code on this for future reference?